docs/en/latest/discovery/nacos.md

```diff
@@ -34,26 +34,45 @@ Add following configuration in `conf/config.yaml` :
 ```yaml
 discovery:
   nacos:
-    host:
+    name: "default"       # Deprecated,see nacos.hosts.name
+    host:                 # Deprecated,see nacos.hosts.host
       - "http://${username}:${password}@${host1}:${port1}"
-    prefix: "/nacos/v1/"
-    fetch_interval: 30    # default 30 sec
-    # `weight` is the `default_weight` that will be attached to each discovered node that
-    # doesn't have a weight explicitly provided in nacos results
-    weight: 100           # default 100
-    timeout:
-      connect: 2000       # default 2000 ms
-      send: 2000          # default 2000 ms
-      read: 5000          # default 5000 ms
+    prefix: "/nacos/v1/"  # Deprecated,see nacos.hosts.prefix.
+    fetch_interval: 30    # default 30 sec.all nacos in config will use this config
+
+    weight: 100           # Deprecated see nacos.hosts.weight
+    timeout:              # Deprecated see nacos.hosts.timeout
+      connect: 2000       # Deprecated see nacos.hosts.timeout
+      send: 2000          # Deprecated see nacos.hosts.timeout
+      read: 5000          # Deprecated see nacos.hosts.timeout
+    access_key: ""        # Deprecated see nacos.hosts.access_key
+    secret_key: ""        # Deprecated see nacos.hosts.secret_key
+    hosts:
+      - name: "your_nacos_cluster_name"  #your nacos cluster name
+        host:
+          - "http://${username}:${password}@${host1}:${port1}"
+        prefix: "/nacos/v1/"
+        # `weight` is the `default_weight` that will be attached to each discovered node that
+        # doesn't have a weight explicitly provided in nacos results
+        weight: 100         # default 100
+        timeout:
+          connect: 2000     # default 2000 ms
+          send: 2000        # default 2000 ms
+          read: 5000        # default 5000 ms
+        access_key: ""        # Nacos AccessKey ID in Alibaba Cloud, notice that it's for Nacos instances on
+        # Microservices Engine (MSE)
+        secret_key: ""        # Nacos AccessKey Secret in Alibaba Cloud, notice that it's for Nacos instances on
+        # Microservices Engine (MSE)
 ```
 
 And you can config it in short by default value:
 
 ```yaml
 discovery:
   nacos:
-    host:
-      - "http://192.168.33.1:8848"
+    hosts:
+      - host:
```

Comment:
this doesn't seem the right way. I think the following is better:

```yaml
nacos:
  hosts:
    - xyz
    - abc
```

Reply:
do you mean  config like this?
```
hosts:
  name1:
    - host:
       - xxx
  name2:
    - host:
       - xxx
```
@shreemaan-abhishek

Reply:
Agree with @ShenFeng312, multiple nacos instances will be used in a multi-cluster environment (with multiple registries), not just for load balancing on a single endpoint.